bundle/Controller/LegacySetupController.php

```diff
@@ -42,31 +42,58 @@ class LegacySetupController implements ContainerAwareInterface
     private $persistenceCachePurger;
 
     /**
-     * @var \Symfony\Component\DependencyInjection\ContainerInterface
+     * @var \eZ\Publish\Core\MVC\Legacy\Kernel\Loader
      */
-    protected $container;
+    protected $kernelFactory;
 
     /**
-     * @var \eZ\Publish\Core\MVC\Legacy\Kernel\Loader
+     * @var \eZ\Bundle\EzPublishLegacyBundle\SetupWizard\ConfigurationDumper
      */
-    protected $kernelFactory;
+    protected $configDumper;
+
+    /**
+     * @var \eZ\Bundle\EzPublishLegacyBundle\SetupWizard\ConfigurationConverter
+     */
+    protected $configConverter;
+
+    /**
+     * @var \Symfony\Component\HttpFoundation\RequestStack
+     */
+    protected $requestStack;
+
+    /**
+     * @var \Symfony\Component\HttpKernel\KernelInterface
+     */
+    protected $kernel;
 
     /**
      * @param \Closure $kernelClosure
      * @param \eZ\Bundle\EzPublishLegacyBundle\DependencyInjection\Configuration\LegacyConfigResolver $legacyConfigResolver
      * @param \eZ\Bundle\EzPublishLegacyBundle\Cache\PersistenceCachePurger $persistenceCachePurger
      * @param \eZ\Publish\Core\MVC\Legacy\Kernel\Loader $kernelFactory
+     * @param \eZ\Bundle\EzPublishLegacyBundle\SetupWizard\ConfigurationDumper $configDumper
+     * @param \eZ\Bundle\EzPublishLegacyBundle\SetupWizard\ConfigurationConverter $configConverter
+     * @param \Symfony\Component\HttpFoundation\RequestStack $requestStack
+     * @param \Symfony\Component\HttpKernel\KernelInterface
```

Comment:
Missed var name in this line?

Reply:
I think the whole PHPDoc could be removed

Reply:
Yep, thanks. Will fix it tomorrow, when I'm at my laptop :)